ui-components/src/main/kotlin/team/duckie/quackquack/ui/component/textfield.kt

```diff
@@ -206,19 +206,19 @@ private inline fun QuackTextFieldDecorationBox(
             }
         },
     ) { measurables, _ ->
-        // The current implementation assumes that the size of the
-        // decoration item is always square. However, the actual size of
-        // the decoration item may not be square. See images below.
+        // The current implementation assumes that the shape of the
+        // decoration item is always square. However, the actual shape of
+        // the decoration item may not be square. For instance, see the images below.
         //
         // [TextField decoration items - square version]
         // https://user-images.githubusercontent.com/40740128/189829870-cba93fd6-d5f4-4016-b826-c6093cfbb386.png
         //
         // [TextField decoration items - rectangular version]
         // https://user-images.githubusercontent.com/40740128/189829922-667b1297-86fa-4e86-9db0-56de1d3ac778.png
         //
-        // However, the height of the decoration item is always the same as
-        // the height of the TextField. Therefore, it is necessary to make
-        // the width of the decoration item to wrap_content.
+        // Regardless of the shape, the height of the decoration item is always the same as
+        // the height of the TextField, as for the width,
+        // it's size is equal to the total length content of the decorated item [wrapped]
```

Comment:
Awesome explanation! You explained `wrap_content` well. But what does the "[wrapped]" at the end mean?

Reply:
wrapped means wrap_content, I should change it to wrap_content instead haha

Reply:
```suggestion
        // it's size is equal to the total length content of the decorated item [wrap_content]
```

Cool!

Reply:
Yeap, I agree with the suggestion , it can be merged now.